README.md

```diff
@@ -186,6 +187,31 @@ method takes one argument:
 
 2. `map` must return a value of the same Functor
 
+### Contravariant Functor
+
+1. `u.contramap(a => a)` is equivalent to `u` (identity)
+2. `u.contramap(x => f(g(x)))` is equivalent to `u.contramap(f).contramap(g)`
+(composition)
+
+#### `contramap` method
+
+```hs
+contramap :: Contravariant f => f a ~> (b -> a) -> f b
```

Comment:
To answer @davidchambers question: yes, I think we should rename this to `Contravariant`. Afterwhich, we should merge this.

@rjmk Are you up to this last change? If not, let me know and I'll make it and get this merged.